### PR TITLE
Create and query database implementation

### PR DESCRIPTION
**Pull request by serengia**
Hello. Kindly review my request to merge `development` branch to `main` branch. 
- The `development` branch contains the creation, insertion, and querying of the database implementation
- The SQL statements for creating the database, inserting into the database, and querying the database are contained in `schema.sql`, `data.sql` and `queries.sql` respectively

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments <a name="acknowledgements"></a>
 
-- The Web App design was inspired by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+- Hat tip to anyone who inspired this project
```
Comment (rotimiazeez) [inline README.md:77]: Please note that the link provided for your MIT License is invalid, kindly fix this 🙏 
Screenshot_20230328_124955

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @serengia 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
### Please ensure you add screenshots of your query results to the pull request description.
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (serengia) [reply README.md:77]: Well noted. Fixing the issue as soon as possible.

**Review by omarsalem7 — APPROVED** [review]
Hi @serengia 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
- The project has met all the technical aspects you needed for this project. Keep rocking!  ✔️


Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] Kindly make sure the next time to open the pull request from the feature branch to the development branch because the next time your project will be invalid

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by serengia** [comment]
Hello. Kindly review my request to marge `revert-1-development` to `main`. The branch contains the implementation of the following features:
- Table updates with `transactions`: BEGIN; COMMIT; CALLBACK; SAVECHECHPOINT 'checkpoint_name';
begin-delete-animals-born-after2022-01-01
---
- Advanced queries with Aggregation pipelines
advanced-queries-with-aggregation

**Pull request merged**